services/121-service/src/fsp-integrations/integrations/intersolve-visa/services/intersolve-visa.service.ts

```diff
@@ -350,16 +336,16 @@ export class IntersolveVisaService {
     return [intersolveVisaChildWallet];
   }
 
-  private async linkChildWalletToParentWalletIfUnlinked(
+  public async linkChildWalletToParentWalletIfUnlinked(
```

Comment:
```suggestion
  private async linkChildWalletToParentWalletIfUnlinked(
```


Does not need to be public anymore, since it's now in linkPhysicalCardToCustomer function